packages/runtime/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@oazapfts/runtime",
+  "version": "0.0.0-development",
+  "description": "Runtime for OpenApi TypeScript client generator",
+  "type": "module",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
+    },
+    "./query": {
+      "types": "./dist/query.d.ts",
+      "require": "./dist/query.cjs",
+      "import": "./dist/query.js"
+    },
+    "./util": {
+      "types": "./dist/util.d.ts",
+      "require": "./dist/util.cjs",
+      "import": "./dist/util.js"
+    },
+    "./headers": {
+      "types": "./dist/headers.d.ts",
+      "require": "./dist/headers.cjs",
+      "import": "./dist/headers.js"
```

I believe these are functionally equivalent, but I've found the error messages can be better with `default` than `import` when people do something wrong
```suggestion
      "require": "./dist/index.cjs",
      "default": "./dist/index.js"
    },
    "./query": {
      "types": "./dist/query.d.ts",
      "require": "./dist/query.cjs",
      "default": "./dist/query.js"
    },
    "./util": {
      "types": "./dist/util.d.ts",
      "require": "./dist/util.cjs",
      "default": "./dist/util.js"
    },
    "./headers": {
      "types": "./dist/headers.d.ts",
      "require": "./dist/headers.cjs",
      "default": "./dist/headers.js"
```